lib/email_tracking_retriever.py

```diff
@@ -290,5 +294,15 @@ def log_in_if_necessary(driver: WebDriver):
   orders_containers = driver.find_elements_by_id('ordersContainer')
   if len(orders_containers) == 0:
     # likely a login screen
+    print("Signing into Amazon ...")
     driver.find_element_by_css_selector('form[name="signIn"]')
-    input('Please log in to an Amazon account on the opened Chrome profile. Hit ENTER when done.')
+    driver.find_element_by_xpath("//input[@type='email']").send_keys(AMAZON['email'])
+    driver.find_element_by_xpath("//input[@type='submit']").click()
+    time.sleep(2)
+    driver.find_element_by_xpath("//input[@type='password']").send_keys(AMAZON['password'])
+    driver.find_element_by_xpath("//input[@type='submit']").click()
+    time.sleep(2)
+
+    two_factor_auth = driver.find_elements_by_id('auth-mfa-otpcode')
```

Comment:
Because of how Selenium timeouts work, calling `find_elements` when the thing doesn't exist will take the full 10 seconds (or whatever the timeout is). It's better to run queries for things that we expect to exist in the common case (or in all cases, then inspect the returned values) so that most of the time we're not waiting those N seconds.